dev/user.clj

```diff
@@ -208,11 +208,11 @@
 
 
   ;; Query Peer
-  (start {:fdb-group-servers       "myserver@localhost:9790"
-          :fdb-group-servers-ports "localhost:8090"
-          :fdb-api-port            8080
+  (start {:fdb-group-servers-ports "localhost:8090" ; ledger api port - this is where we should be forwarding our qs and txs
```

Comment:
This config param (and how it interacts with `:fdb-group-servers`) is so confusing and overloaded at this point, I'm tempted to say we should deprecate it and go back to the drawing board. What do you think?

Reply:
Either way, this param or whatever we replace it with will also need to be added to the canonical list of config params in the `server-settings` namespace to work outside of a REPL.

Reply:
👍🏻

Renaming it might be sufficient, except for the role it plays in transactors (the specifics of which I can never remember haha). Might be better to create a new one for this like `:fdb-query-upstream-transactor`?